scripts/android/build.sh

```diff
@@ -1,5 +1,7 @@
-#!/bin/bash
-mv android-5.0 android-5.0_source && mkdir -p android-5.0/platforms/android-21 && cp -r android-5.0_source/* android-5.0/platforms/android-21/ && rm -rf android-5.0_source
+#!/bin/bash -e
+mv android-9 android-9_source && mkdir -p android-9/platforms/android-28 && cp -r android-9_source/* android-9/platforms/android-28/ && rm -rf android-9_source
 mkdir -p build && cd build
-cmake .. -DCMAKE_TOOLCHAIN_FILE="$WORKSPACE/android-ndk-r19c/build/cmake/android.toolchain.cmake" -DANDROID_ABI=arm64-v8a -DEDGE_SDK_ENABLE_TESTING=NO
-make
+cmake .. -DCMAKE_TOOLCHAIN_FILE="$WORKSPACE/android-ndk-r20/build/cmake/android.toolchain.cmake" -DANDROID_PLATFORM=android-28 -DANDROID_ABI=arm64-v8a -DEDGE_SDK_ENABLE_TESTING=NO -DEDGE_SDK_BUILD_EXAMPLES=ON
+sudo make install -j16
+cd examples/dataservice-read/android
```

Comment:
The same steps should be performed for dataservice-write example:
```suggestion
pushd .
cd examples/dataservice-read/android
sudo ./gradlew assemble
popd
cd examples/dataservice-write/android
sudo ./gradlew assemble
```

Alreanatively you can automate this process by traverse through all the folders in the `examples` and invoke `gradlew` command there - this will atuomatically build all upcoming examples without changing the code.